components/mosart/cime_config/testdefs/testmods_dirs/mosart/sediment/shell_commands

```diff
@@ -0,0 +1,51 @@
+./xmlchange LND_DOMAIN_FILE=domain.lnd.nldas2_0224x0464_c110415.nc
+./xmlchange LND_DOMAIN_PATH=/compyfs/inputdata/share/domains/domain.clm
```

Comment:
LND_DOMAIN_PATH would only work for Compy. Please use $DIN_LOC_ROOT/share/domains/domain.clm